src/pages/bounties/[id]/[[...slug]].tsx

```diff
@@ -633,6 +627,46 @@ const useStyles = createStyles((theme) => ({
   },
 }));
 
+const BountyEntries = ({ bounty }: { bounty: BountyGetById }) => {
+  const theme = useMantineTheme();
+  const entryCreateUrl = `/bounties/${bounty.id}/entries/create`;
+
+  return (
+    <Stack spacing="xl" py={32}>
+      <Group position="apart">
+        <Title order={2} size={28} weight={600}>
+          Hunters
+        </Title>
+        <Button size="xs">Submit</Button>
+      </Group>
+      <Paper
+        p="xl"
+        radius="sm"
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+          background: theme.colorScheme === 'dark' ? theme.colors.dark[5] : theme.colors.gray[1],
+        }}
+        component={NextLink}
+        href={entryCreateUrl}
+      >
+        <Stack spacing="sm" align="center">
+          <Text size={24} weight={600} align="center">
+            No submissions yet
+          </Text>
+          <Text color="dimmed" align="center">
+            Be the first to submit your solution.
+          </Text>
+          <Button component={NextLink} href={entryCreateUrl} size="sm" w="75%">
+            Submit
+          </Button>
```

Comment:
Since we can have muted users, we probably don't want them submitting bounty entries. Here's what I would do
```suggestion
          {!currentUser.isMuted && (
            <>
              <Text color="dimmed" align="center">
                Be the first to submit your solution.
              </Text>
              <Button component={NextLink} href={entryCreateUrl} size="sm" w="75%">
                Submit
              </Button>
            </>
          )}
```